managers/paranet-operations-manager.js

```diff
@@ -0,0 +1,384 @@
+const {resolveUAL, getOperationStatusObject} = require('../services/utilities.js');
+const { ethers } = require('ethers');
+
+class ParanetOperationsManager {
+    constructor(services) {
+        this.blockchainService = services.blockchainService;
+        this.inputService = services.inputService;
+        this.nodeApiService = services.nodeApiService;
+        this.validationService = services.validationService;
+    }
+
+    async create(UAL, options = {}) {
+        const {
+            blockchain,
+            paranetName,
+            paranetDescription,
+        } = this.inputService.getParanetCreateArguments(options);
+
+        this.validationService.validateParanetCreate(
+            UAL,
+            blockchain,
+            paranetName,
+            paranetDescription,
+        );
+
+        const { contract, tokenId } = resolveUAL(UAL);
+
+       await this.blockchainService.registerParanet({
+                contract,
+                tokenId,
+                paranetName,
+                paranetDescription,
+            },
+            blockchain
+        );
+
+        return {
+            paranetUAL: UAL,
+        };
+    }
+
+    async deployIncentivesContract(paranetUAL, incentiveType, options = {}) {
+        const {
+            blockchain,
+            tracToNeuroEmissionMultiplier,
+            operatorRewardPercentage,
+            incentivizationProposalVotersRewardPercentage,
+        } = this.inputService.getParanetDeployIncentivesContractArguments(options);
+
+        this.validationService.validateDeployIncentivesContract(
+            paranetUAL,
+            blockchain,
+            tracToNeuroEmissionMultiplier,
+            operatorRewardPercentage,
+            incentivizationProposalVotersRewardPercentage,
+        );
+        if(incentiveType === 'Neuroweb') {
+            const {contract, tokenId} = resolveUAL(paranetUAL);
+
+            await this.blockchainService.deployNeuroIncentivesPool({
+                    contract,
+                    tokenId,
+                    tracToNeuroEmissionMultiplier,
+                    operatorRewardPercentage,
+                    incentivizationProposalVotersRewardPercentage,
+                },
+                blockchain
+            );
+
+            const paranetId = ethers.keccak256(
+                ethers.solidityPacked(['address', 'uint256'], [contract, tokenId]),
+            );
+
+            const neuroIncentivesPoolAddress = await this.blockchainService.getNeuroIncentivesPoolAddress(paranetId, blockchain);
```

Comment:
We should check if we got address not null or 0x0 not to burn coins if error happens

Reply:
That sending tokens were testing part, it is removed